doc/install/installation.md

```diff
@@ -0,0 +1,346 @@
+
+    sudo -u huginn -H bundle exec rake assets:precompile RAILS_ENV=production
+
+### Start Your Huginn Instance
+
+    sudo start huginn
+
+## 7. Nginx
+
+**Note:** Nginx is the officially supported web server for Huginn. If you cannot or do not want to use Nginx as your web server, the wiki has a page on how to configure [apache](https://github.com/cantino/huginn/wiki/Apache-Huginn-configuration).
+
+### Installation
+
+    sudo apt-get install -y nginx
+
+### Site Configuration
+
+Copy the example site config:
+
+    sudo cp lib/support/nginx/huginn /etc/nginx/sites-available/huginn
+    sudo ln -s /etc/nginx/sites-available/huginn /etc/nginx/sites-enabled/huginn
+
+Make sure to edit the config file to match your setup, if you are running multiple nginx sites remove the `default_server` argument from the `listen` directives:
+
+    # Change YOUR_SERVER_FQDN to the fully-qualified
+    # domain name of your host serving Huginn.
+    sudo editor /etc/nginx/sites-available/huginn
+
+Remove the default nginx site, **if huginn is the only enabled nginx site**:
+
+    sudo rm /etc/nginx/sites-enabled/default
+
+**Note:** If you want to use HTTPS, replace the `huginn` Nginx config with `huginn-ssl`. See [Using HTTPS](#using-https) for HTTPS configuration details.
+
+### Test Configuration
+
+Validate your `huginn` or `huginn-ssl` Nginx config file with the following command:
+
+    sudo nginx -t
+
+You should receive `syntax is okay` and `test is successful` messages. If you receive errors check your `huginn` or `huginn-ssl` Nginx config file for typos, etc. as indicated in the error message given.
+
+### Restart
+
+    sudo service nginx restart
+
+# Done!
+
+### Initial Login
+
+Visit YOUR_SERVER in your web browser for your first Huginn login. The setup has created a default admin account for you. You can use it to log in:
+
+    admin
+    password
+
+
+**Enjoy!**
+
+You can use `sudo start huginn` and `sudo stop huginn` to start and stop Huginn.
+
+## Advanced Setup Tips
+
+### Using HTTPS
+
+To use Huginn with HTTPS:
+
+1. In `.env`:
+    1. Set the `FORCE_SSL` option to `true`.
+1. Use the `huginn-ssl` Nginx example config instead of the `huginn` config:
+    1. `sudo cp lib/support/nginx/huginn-ssl /etc/nginx/sites-available/huginn`
+    1. Update `YOUR_SERVER_FQDN`.
+    1. Update `ssl_certificate` and `ssl_certificate_key`.
+    1. Review the configuration file and consider applying other security and performance enhancing features.
+
+Restart Nginx, export the init script and restart Huginn:
+
+```
+cd /home/huginn/huginn
+sudo service nginx restart
+sudo foreman export upstart -a huginn /etc/init
+sudo restart huginn
+```
+
+Using a self-signed certificate is discouraged, but if you must use it follow the normal directions. Then generate the certificate:
+
+```
+sudo mkdir -p /etc/nginx/ssl/
+cd /etc/nginx/ssl/
+sudo openssl req -newkey rsa:2048 -x509 -nodes -days 3560 -out huginn.crt -keyout huginn.key
+sudo chmod o-r huginn.key
+```
```

Comment:
We should have a section on how to update your copy of Huginn and re-deploy it too.